openapi3/swagger_loader.go

```diff
@@ -145,17 +146,26 @@ func (swaggerLoader *SwaggerLoader) ResolveRefsIn(swagger *Swagger, path *url.UR
 	}
 
 	// Visit all operations
-	for _, pathItem := range swagger.Paths {
+	for endpoint, pathItem := range swagger.Paths {
 		if pathItem == nil {
 			continue
 		}
-		for _, parameter := range pathItem.Parameters {
+		// Refer on documentation at Open Api Specification 3.0 at
+		// https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#pathItemObject
+		// there is one property is $ref, to get the configuration at another reference object. For this time this issue
+		// will be fix, with the rules the value just come from relative path file or network file
```

Comment:
I think it would be better to move this out of this loop into its own loop, right before. This way paths get resolved first, then their contents get resolved.
Also can you remove your comments?